regolith/helpers/makeappointmentshelper.py

```diff
@@ -17,18 +17,26 @@
 from regolith.tools import (
     all_docs_from_collection,
     get_pi_id,
+    fuzzy_retrieval,
+    is_fully_appointed
+)
+from regolith.dates import (
+    is_current,
+    has_finished,
 )
-from regolith.dates import is_current
 
 TARGET_COLL = "people"
 HELPER_TARGET = "makeappointments"
 
 
 def subparser(subpi):
 
+    subpi.add_argument("--date", help="date to check appointments and grants against")
```

Comment:
this help message is a bit cryptic to me....when would I use date and why?  I can see the begin_date and range logic

Reply:
Hmm, I'll make it clear that the `begin_date` and `end_date` are specified when the user wants to check for an interval of time, and that the `desired_date` (instead of simply 'date') is specified when the user wants to check if an appointment is -something- on a specific date

Reply:
I don't think we need the second thing.  When would I ever want to know if someone is fully loaded on a single day?    I guess if we wanted to call it in a loop or something?  But from the command line?  I think I am inclined to remove that and we can add it back later if it is really needed, but I suspect it won't be.....   We use here the "keep it simple" rule

Reply:
done